aicsimageio/readers/reader.py

```diff
@@ -175,42 +172,76 @@ def current_scene_index(self) -> int:
         scene_index: int
             The current operating scene index in the file.
         """
-        return self.scenes.index(self.current_scene)
+        return self._current_scene_index
+
+    def _reset_self(self) -> None:
```

Comment:
Maybe just `_reset`?

Reply:
Nah. I am very partial to `_reset_self` here for some reason.